x-pack/plugins/canvas/public/feature_catalogue_entry.ts

```diff
@@ -15,6 +15,6 @@ export const featureCatalogueEntry = {
   }),
   icon: 'canvasApp',
   path: '/app/canvas',
-  showOnHomePage: true,
+  showOnHomePage: false,
```

Comment:
😢

Reply:
There is a new Kibana landing page coming alongside this page that will highlight Canvas!

Reply:
👏